packages/core/src/login/oidc/__mocks__/IncomingRedirectHandler.ts

```diff
@@ -22,23 +22,17 @@
 import { EventEmitter } from "events";
 import { jest } from "@jest/globals";
 
-import IIncomingRedirectHandler, {
-  IncomingRedirectInput,
-  IncomingRedirectResult,
-} from "../IIncomingRedirectHandler";
+import IIncomingRedirectHandler from "../IIncomingRedirectHandler";
 
-const canHandle = jest.fn<Promise<boolean>, IncomingRedirectInput>(
-  (_url: string) => Promise.resolve(true)
-);
+const canHandle = jest.fn((_url: string) => Promise.resolve(true));
 
-const handle = jest.fn<Promise<IncomingRedirectResult>, IncomingRedirectInput>(
-  (_url: string, _emitter: EventEmitter | undefined) =>
-    Promise.resolve({
-      sessionId: "global",
-      isLoggedIn: true,
-      webId: "https://pod.com/profile/card#me",
-      fetch: jest.fn(),
-    })
+const handle = jest.fn((_url: string, _emitter: EventEmitter | undefined) =>
+  Promise.resolve({
+    sessionId: "global",
+    isLoggedIn: true,
+    webId: "https://pod.com/profile/card#me",
```

Comment:
```suggestion
    webId: "https://webid.test/profile/card#me",
```

We shouldn't use real URLs in tests, `pod.com` is potentially real

Reply:
I'll add a tech debt ticket to do this all at once, it applies across all codebase